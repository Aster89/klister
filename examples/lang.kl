#lang "prelude.kl"

[define f [lambda (x y z) y]]

[example (f [quote æ] [quote ø] [quote å])]
