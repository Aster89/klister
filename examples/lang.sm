#lang "n-ary-app.sm"

[define f [lambda (x y z) y]]

[example (f [quote æ] [quote ø] [quote å])]
