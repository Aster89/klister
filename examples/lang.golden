#[/Users/gelisam/working/haskell/klister/examples/lang.kl:5.30-5.31]<Ã¸> : Syntax
